src/main/webapp/js/instructorFeedbackResults.es6

```diff
@@ -69,39 +79,39 @@ $(document).ready(function() {
         $('#mainContent').printThis({
             importCSS: true,
             importStyle: true,
-            loadCSS: '/stylesheets/printview.css'
+            loadCSS: '/stylesheets/printview.css',
         });
     });
 });
 
 function submitFormAjax() {
-    var formObject = $('#csvToHtmlForm');
-    var formData = formObject.serialize();
-    var content = $('#fsModalTable');
-    var ajaxStatus = $('#ajaxStatus');
+    const formObject = $('#csvToHtmlForm');
+    const formData = formObject.serialize();
+    const content = $('#fsModalTable');
+    const ajaxStatus = $('#ajaxStatus');
     $.ajax({
         type: 'POST',
-        url: '/page/instructorFeedbackResultsPage?' + formData,
-        beforeSend: function() {
+        url: `/page/instructorFeedbackResultsPage?${formData}`,
+        beforeSend() {
             content.html('<img src="/images/ajax-loader.gif">');
         },
-        error: function() {
+        error() {
             ajaxStatus.html('Failed to load results table. Please try again.');
             content.html('<button class="btn btn-info" onclick="submitFormAjax()"> retry</button>');
         },
-        success: function(data) {
-            setTimeout(function() {
+        success(data) {
+            setTimeout(() => {
                 if (data.isError) {
                     ajaxStatus.html(data.errorMessage);
                     content.html('<button class="btn btn-info" onclick="submitFormAjax()"> retry</button>');
                 } else {
-                    var table = data.sessionResultsHtmlTableAsString;
-                    content.html('<small>' + table + '</small>');
+                    const table = data.sessionResultsHtmlTableAsString;
+                    content.html(`<small>${table}</small>`);
                     ajaxStatus.html(data.ajaxStatus);
                 }
                 setStatusMessage(data.statusForAjax);
             }, 500);
-        }
+        },
     });
 }
 
```

Comment:
Another one for future PR: it won't let me insert this comment inside the actual function, but showHideStats should be renamed to e.g toggleStatsDisplay.